st2common/st2common/content/bootstrap.py

```diff
@@ -248,6 +252,28 @@ def register_policies():
     LOG.info('Registered %s policies.', registered_count)
 
 
+def register_configs():
+    pack_dir = cfg.CONF.register.pack
+    fail_on_failure = cfg.CONF.register.fail_on_failure
+
+    registered_count = 0
+
+    try:
+        LOG.info('=========================================================')
+        LOG.info('############## Registering configs ######################')
+        LOG.info('=========================================================')
+        registered_count = configs_registrar.register_configs(pack_dir=pack_dir,
+                                                              fail_on_failure=fail_on_failure)
+    except Exception as e:
+        exc_info = not fail_on_failure
+        LOG.warning('Failed to register configs: %s', e, exc_info=exc_info)
+
+        if fail_on_failure:
+            raise e
+
+    LOG.info('Registered %s configs.' % (registered_count))
```

Comment:
Sigh, always getting distracted by minor details (which were actually copied, for better or worse) from other registrar classes :P